torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -2523,6 +2590,14 @@ std::unordered_map<kir::IterDomain*, kir::Val*> getPredicateReferenceIndexing(
       std::inserter(loop_to_ind_map, loop_to_ind_map.begin()),
       [](kir::ForLoop* fl) { return std::make_pair(fl, fl->index()); });
 
+  // We shouldn't have any predicates within mma loop, and this
+  //  is just a temporary work around to make these predicates compile
+  for (auto& it : loop_to_ind_map) {
```

Comment:
This is a temporary workaround. We will not have any predicates within a "mma loop" eventually so this can be removed later.